packages/structure/src/schema/AttributeDefinitions/AttributeDefinition.js

```diff
@@ -62,7 +62,7 @@ class AttributeDefinition {
 
     this.name = name;
     this.options = options;
-    this.dynamicDefault = isFunction(options.default);
+    this.hasDynamicDefault = isFunction(options.default);
```

Comment:
I wonder if we should call this as `isDynamicDefault`, I'm thinking something like, we may or may not have a default attribute and it may or may not be dynamic.

 `hasDefault` and `isDynamicDefault`

What do you think?

Reply:
:thinking: You have a point, I'm gonna change!